src/main/java/com/github/dockerjava/api/command/UpdateContainerCmd.java

```diff
@@ -0,0 +1,68 @@
+package com.github.dockerjava.api.command;
+
+import com.github.dockerjava.api.model.UpdateContainerResponse;
+
+import javax.annotation.CheckForNull;
+
+/**
+ * @author Kanstantsin Shautsou
+ */
+public interface UpdateContainerCmd extends SyncDockerCmd<UpdateContainerResponse> {
+    @CheckForNull
+    String getContainerId();
+
+    @CheckForNull
+    public Integer getBlkioWeight();
+
+    public UpdateContainerCmd withBlkioWeight(Integer blkioWeight);
+
+    public UpdateContainerCmd withContainerId(String containerId);
```

Comment:
as low level api just send to null container id (btw, can container be reached via null name?). Buy yeah, while it ~CLI interface for library users, then makes sense.